Pet-Clinic-Frontend/src/pages/Login.tsx

```diff
@@ -36,9 +38,10 @@ const Login = () => {
       });
       if (!response.status.toString().startsWith('2')) {
         setError('Invalid email or password');
+      } else {
+        login()
```

Comment:
Well this login is a bit counter intuitive for me. Shouldn't we store the user somehow here? Maybe it is not implemented yet, but we should not just "say" that we are logged in. We should store something at least.

Reply:
I don't know, maybe. It is ok for me at least now.  I would rather discuss this in class channel tomorrow.